tests/errors/SettingsError/output/o01.conf

```diff
@@ -0,0 +1,26 @@
+inputs += o01.ledger
+
+reportOptions = {
+  isRight = [Liability, Capital, Income]
+}
+
+// overwriting o01.ledger should result in an error
+reports += {
+  title = "Balance Sheet"
+  type = balance
+  outFiles = [o01.ledger, "-"]
+  accountMatch = ["^Assets.*", "^Liability.*", "^Capital.*"]
+}
+
+reports += {
+  title = "Profit and Loss" 
+  type = balance
+  accountMatch = ["^Income.*", "^Expenses.*"]
+}
+
+// overwriting o01-include.ledger should result in an error
+exports += {
+  type = balance
+  format = xml
+  outFiles = [o01-include.ledger]
```

Comment:
I think these tests should be split. Otherwise, the error from "overwriting o01.ledger" will mask the absence of other errors.